src/components/People.js

```diff
@@ -0,0 +1,74 @@
+import React, { Component } from "react";
+import axios from "axios";
+
+export default class People extends Component {
+  constructor() {
+    super();
+    this.state = {
+      data: [],
+      input: "",
+      name: "",
+      age: "",
+      gender: "",
+      errPing: "",
+    };
+  }
+  handleInputChange = (e) => {
+    this.setState({
+      input: e.target.value,
+    });
+    console.log(this.state.input);
+  };
+
+  handleClick = async () => {
+    try {
+      const { data } = await axios.get(
+        `https://ghibliapi.herokuapp.com/people`
+      );
+      console.log(data);
+
+      const uuid = data.find((person) => person.name === this.state.input);
+      console.log(uuid);
+      this.setState({
+        name: uuid.name,
+        age: uuid.age,
+        gender: uuid.gender,
+      });
+    } catch (e) {
+      console.error(e);
+
+      this.setState({
+        name: "",
+        age: "",
+        gender: "",
+        errPing: "Not Found",
+      });
+    }
+  }; 
+
+  render() {
+    return (
+      <div>
+        <h1>Search for a Person</h1>
+        <input
```

Comment:
The `input` tag should be a controlled by react. see my comment on MovieGrabber for more info.
The cypress tests for the placeholder are case sensitive!